src/AcmTable/AcmTable.tsx

```diff
@@ -834,7 +834,10 @@ export function AcmTable<T>(props: AcmTableProps<T>) {
                 props.emptyState ? (
                     <PageSection padding={{ default: 'noPadding' }}>{props.emptyState}</PageSection>
                 ) : (
-                    <PageSection padding={{ default: 'noPadding' }}>
+                    <PageSection
+                        variant={props.extraToolbarControls ? 'light' : 'default'}
+                        padding={{ default: 'noPadding' }}
+                    >
```

Comment:
@richiepiya This seems to only cover the case where we are using the default `<AcmEmptyState>`. In app-ui, we pass in our own `<AcmEmptyState>` via the `emptyState`.

Reply:
Can you see any reason why the `PageSection` is repeated here? Maybe you could combine them into one, and just choose which empty state to use inside the `PageSection`?

Reply:
fixed, thanks!